adding Linux build support

maker99 opened the pull request:
adding Linux build support

maker99 committed:
Update Makefile

maker99 committed:
Create gen_timestamp.sh

for Linux build support

maker99 committed:
Update smith.h

fixing case for include LCD.h

chrta commented:
See https://bitbucket.org/kuchura/eu1ky_aa_v3/pull-requests/6/linux-support/diff

maker99 committed:
allow building firmware under Linux with Github actions (#2)

* Create build.yml

* Rename .github/build.yml to .github/workflows/build.yml

* fixing linux builds

* changing Wiki to github.com/EU1KY/eu1ky_aa_v3/wiki

* Remove unused camera driver.

uplifted from https://github.com/EU1KY/eu1ky_aa_v3/commit/2f533b1afaa07fbfd285a11ad35637bcbee1e20c
@chrta, chrta committed on May 11, 2018 
1 parent 08a70c0 commit 2f533b1afaa07fbfd285a11ad35637bcbee1e20c

* Update Makefile

Co-authored-by: maker99 <maker99@noreply.github.com>

maker99 committed:
changing github action trigger

will now trigger automated firmware build when pushing to maker99-Linux_support
